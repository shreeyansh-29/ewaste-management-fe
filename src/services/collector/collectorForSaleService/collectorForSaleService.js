import {COLLECTOR_SELL} from "../../../container/constant/constants";
import api from "../../../core/utilities/httpProvider";

export const collectorForSaleService = (data) => {
  return api.post(COLLECTOR_SELL, {
    itemName: data.payload.itemName,
    category: data.payload.category,
    quantity: data.payload.quantity,
    price: data.payload.price,
    status: data.payload.status,
  });
};
