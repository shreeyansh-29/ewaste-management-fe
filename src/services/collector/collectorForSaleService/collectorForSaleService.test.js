/* eslint-disable no-undef */

import {collectorForSaleService} from "./collectorForSaleService";
import React from "react";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing collectorForSaleService", () => {
  const data = {payload: {category: "", itemName: "", quantity: "", price: ""}};
  const Service = collectorForSaleService;
  const wrapper = shallow(<Service {...data} />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
