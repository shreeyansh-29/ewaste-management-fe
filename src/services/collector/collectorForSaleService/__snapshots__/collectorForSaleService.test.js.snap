// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing collectorForSaleService 1`] = `<Component />`;
